lighthouse/blueprints/plates.py

```diff
@@ -13,7 +13,7 @@
 from lighthouse.constants.fields import FIELD_PLATE_BARCODE
 from lighthouse.helpers.general import get_fit_to_pick_samples_and_counts
 from lighthouse.helpers.plates import (
-    add_cog_barcodes_from_different_centres,
```

Comment:
Is this method needed then?

Reply:
Yes, is in use for Beckman currently, but it won't be needed for Sentinel.

Reply:
👍🏼